doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nRF54H20-1-rc2.rst

```diff
@@ -1,7 +1,7 @@
-.. _ncs_release_notes_290-nrf54h20-rc1:
+.. _ncs_release_notes_290-nrf54h20-1-rc2:
 
-|NCS| v2.9.0-nRF54H20-rc1 Release Notes
-#######################################
+|NCS| v2.9.0-nRF54H20-1-rc2 Release Notes
```

Comment:
If we are treating rc1 and rc2 as two different releases, shouldn't we keep both release notes instead of replacing the files?
Applies for other files as well.